modelgym/util.py

```diff
@@ -62,3 +73,72 @@ def split_and_preprocess(X_train, y_train, X_test, y_test, cat_cols=[], n_splits
     full_dtest = XYCDataset(X_test.astype(float), y_test, cat_cols)
 
     return cv_pairs, (full_dtrain, full_dtest)
+
+
+def process_params_gp(model1):
```

Comment:
I'd name this functions something like "hyperopt2skopt_space".. (or "hyperspace2skopt") it would be much more clear that "preprocess"

Reply:
great idea